packages/components/src/theme/groups.scss

```diff
@@ -0,0 +1,79 @@
+
+.group-save-alert {
```

Comment:
A lot of these styles are essentially the same as in permissions.scss, but with the names changed. Wasn't sure whether to opt for a new style file, with new classnames for automated test selector ease, or to refactor the names of classes in permissions.scss to also be applicable to groups to avoid maintaining two similar stylesheets.

Reply:
I would opt for re-using classes where we can instead of duplicating the styles here. That way if we change the styles in the future it's much easier.

As for making it easier to test, you can still add group-specific class names where you render group stuff that way selenium can more easily find things, you don't need to having matching CSS selectors for those classes.